client/src/components/Player.js

```diff
@@ -28,24 +28,44 @@ const PlayerMain = styled.div`
 `
 
 class Player extends Component {
```

Comment:
As I said before, code consistency is very important thing, so if you decided to go with `fat arrow function` stick to it, do not mix with `old fashion` ones. 

> NOTE: constructor() is exception, it cannot be written in `fat arrow` way.